__init__.py

```diff
@@ -23,27 +23,18 @@
 def get_version_string():
     return str(bl_info['version'][0]) + '.' + str(bl_info['version'][1]) + '.' + str(bl_info['version'][2])
 
-# Monkey patch to add gather_gltf_hook, must be removed once handled upstream https://github.com/KhronosGroup/glTF-Blender-IO/issues/1009
+# gather_gltf_hook does not expose the info we need, make a custom hook for now
+# ideally we can resolve this upstream somehow https://github.com/KhronosGroup/glTF-Blender-IO/issues/1009
 from io_scene_gltf2.blender.exp import gltf2_blender_export
 from io_scene_gltf2.io.exp.gltf2_io_user_extensions import export_user_extensions
 orig_gather_gltf = gltf2_blender_export.__gather_gltf
 def patched_gather_gltf(exporter, export_settings):
     orig_gather_gltf(exporter, export_settings)
-    export_user_extensions('gather_gltf_hook', export_settings, exporter._GlTF2Exporter__gltf)
+    export_user_extensions('hubs_gather_gltf_hook', export_settings, exporter._GlTF2Exporter__gltf)
```

Comment:
A `gather_gltf_hook` was added upstream https://github.com/KhronosGroup/glTF-Blender-IO/commit/0827c6d857d2156b09375eef9cefddf5924fb5c8#diff-37cfcca6b43542789078e990ff811ea4 but it does not expose the attributes we actually need, so continue to apply a monkey patch. Discussion about this happening here https://github.com/KhronosGroup/glTF-Blender-IO/issues/1009

Reply:
fwiw if we will be moving towards `s/hubs/moz`(as on #3 ) could it be better to use `moz_gather_gltf_hook`?

Reply:
Yeah I figure it would be best to just do that all at once in #3 so we can see it all in one spot. I think I will tackle that alongside reworking how settings are configured.